rules/consistent-code-style/rule.md

```diff
@@ -15,33 +15,41 @@ guid: 292e4a99-4aa0-45f4-a9de-18ba352f9a68
 
 It's important that the code in a project is kept consistent. This is hard to do when you have developers working in different environments. 
 
-Using a .editorconfig file is the best way to manage this.
+Using a *.editorconfig* file is the best way to manage this.
 
 <!--endintro-->
 
-The EditorConfig file specification is defined here https://editorconfig-specification.readthedocs.io/
+See the [EditorConfig file specification](https://spec.editorconfig.org)
 
-Most IDEs will automatically find and use a .editorconfig file to format code.
-See https://devblogs.microsoft.com/visualstudio/visual-studio-17-1-preview-2-is-now-available/#keep-your-code-clean-automatically
+Most IDEs will automatically find and use a *.editorconfig* file to format code.
+
+See [Keep your code clean, automatically!](https://devblogs.microsoft.com/visualstudio/visual-studio-17-1-preview-2-is-now-available/#keep-your-code-clean-automatically).
 
 ::: good
-![Good Example - Project using a .editorconfig file](vs2022-add-editorconfig.png)
+![Good example - Project using a ".editorconfig" file](vs2022-add-editorconfig.png)
 :::
 
 ::: bad
-![Bad Example - Project using StyleCop (old)](vs-2022-stylecop.png)
+![Bad example - Project using StyleCop (old)](vs-2022-stylecop.png)
 :::
 
-## Creating .editorconfig files
+## Creating **.editorconfig** files
 
 **In VS 2022**
 1. Open the Add New Item dialog (Ctrl+Shift+A)
-1. Search for "EditorConfig"
-1. Select a config file depending on your project
+2. Search for "EditorConfig"
+3. Select a config file depending on your project
+
 ![Figure: Creating .editorconfig in VS 2022](vs2022-add-editorconfig.png)
 
```

Comment:
Hi Brady

Your instructions are a bit on the skinny side. I'd like to see more about how to configure a good coding style and preserve it via .editorConfig and then sharing it with the team for consistency.

e.g 

To add an editor config, you can use:
- VS2022 to add a new `.editorConfig` with some of the .NET defaults as you described already
- Dotnet CLI (same defaults). 

In VS2022 you can configure your code style by going to Tools | Options | Text Editor | C# | Code Style 
From here you can generate a .editorConfig as well with all the custom VS rules that aren't part of the regular .editorConfig spec and still share them with the team.

![image](https://user-images.githubusercontent.com/23464439/236663697-f798905d-c5ac-4035-bf88-1e7642e756be.png)


I'd also note that you can use `dotnet format` to apply all the whitespace, code style and code analysis changes to the project. Or you can  run them separately as:

- `dotnet format whitespace`
- `dotnet format style`
- `dotnet format analyzers`